src/import/import-helper.js

```diff
@@ -47,6 +47,9 @@ export async function runImportJobAndPoll( {
   // Determine the base URL
   const baseURL = getApiBaseUrl(stage);
 
+  // Filter out obviously invalid URLs, and ignore comments.
+  const filteredUrls = urls && Array.isArray(urls) ? urls.filter((url) => url.length > 4 && url[0] !== '#') : [];
```

Comment:
Can be simplified a bit:

```suggestion
  const filteredUrls = Array.isArray(urls) ? urls.filter((url) => url.length > 4 && url[0] !== '#') : [];
```